.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,5 +1,5 @@
 name: Bug report
-title: "[Bug] "
+title: "[<Ray Component Here>] "
```

Comment:
Decision: Use the title to tag components. This is redundant with label sometimes, but good for clarity.

Reply:
Suggestion to add examples like "core|tune|..."